src/webapp/reports/validate-custom-forms/ValidateCustomFormsReport.tsx

```diff
@@ -0,0 +1,99 @@
+import { Typography, makeStyles } from "@material-ui/core";
+import _ from "lodash";
+import React, { useState } from "react";
+import i18n from "../../../locales";
+import { Spinner } from "../../components/objects-list/Spinner";
+import { Select, SelectOption } from "../../components/select/Select";
+import { useAppContext } from "../../contexts/app-context";
+
+const list = [{ text: "" }];
+
+const ValidateCustomFormsReport: React.FC = () => {
+    const [isLoading, setLoading] = useState(false);
+
+    const { compositionRoot, config } = useAppContext();
+    const OnModuleChange = async ({ value }: SelectOption) => {
+        setLoading(true);
+        const result = await compositionRoot.validateCustomForm.get(value);
+        _.remove(list);
+        if (result.length === 0) {
+            list.push({ text: "No errors detected" });
+        }
+        result.map(item => {
+            return list.push({ text: item });
+        });
+        setLoading(false);
+        return value;
+    };
+
+    const classes = useStyles();
+
+    const [modules] = React.useState<{ value: string; label: string }[]>(
```

Comment:
Something is wrong here. If you don't use the second item in the response (the setter), you don't have a state but a static value.

Reply:
I have remove the useState